### PR TITLE
TA-1284 Test implementation

### PR DESCRIPTION
**Pull request by EvgeniiNikonets**

### CODE REVIEW

File: test_suites/content_filter_profile_test_max_length.json [hunk 1]
```diff
@@ -0,0 +1,1587 @@
...
+                "Value": "test-site"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "Name": "Max length Requests",
+      "Skip": false,
```
Comment (olgasobolevsky) [inline test_suites/content_filter_profile_test_max_length.json:642]: Rename this test:
"Name": "Create content filter profiles for Max length Requests"
And split it to 4 Tests:
1. "Name": "Create content filter profiles for Max length Requests" : in this test create all the content filters including the filter for step 17.
2. "Name": "Setup security policy, proxy template and server group" : in this test create security policy, proxy template for step 17 and server group that use this security policy and proxy template. Publish.
3. "Name":"Send traffic" : in this test send traffic including the step 17.
4. "Name":"Assert logs" : assert logs
5. "Name":"Max body limit smaller than Arguments max length": in this test change the proxy template  for step 18, publish, send traffic.

File: test_suites/content_filter_profile_test_max_length.json [hunk 2]
```diff
@@ -0,0 +1,1587 @@
...
+            },
+            {
+              "Key": "test",
+              "Value": "testqFTuLBoYDxvEplgJhXnKfZScmOaNPQrwHzRVtjAdCeIsyGUE4aBvRcsdfs"
+            }
+          ],
+          "Expected": {
+            "Code": 473,
+            "Type": "status-code"
+          }
```
Comment (olgasobolevsky) [inline test_suites/content_filter_profile_test_max_length.json:1061]: For the consistency, change the Expected to same style as on other requests:
 "Expected": {
                        "Code": "473",
                        "Type": "content",
                        "Single": true,
                        "Text": "b''"
                    }

File: example_suites/content_filter_profile_test_max_length.json [hunk 3]
```diff
@@ -0,0 +1,1752 @@
...
+          },
+          "Method": "GET",
+          "Headers": [
+            {
+              "Key": "Host",
+              "Value": "test.site"
+            },
+            {
+              "Key": "test",
+              "Value": "testqFTuLBoYDxvEplgJhXnKfZScmOaNPQrwHzRVtjAdCeIsyGUE4aBvR12"
```
Comment (DmytroBalkovyi) [inline example_suites/content_filter_profile_test_max_length.json:1139]: actual length of value is 59 instead of 60

Comment (olgasobolevsky) [reply example_suites/content_filter_profile_test_max_length.json]: fixed

**Review by olgasobolevsky — APPROVED** [review]

**Pull request merged**